metadata.json

```diff
@@ -82,7 +82,7 @@
   "dependencies": [
     {"name":"ghoneycutt/common","version_requirement":">= 1.0.3"},
     {"name":"leinaddm/htpasswd","version_requirement":">= 0.0.1"},
-    {"name":"puppetlabs/stdlib","version_requirement":">= 3.2.0"},
+    {"name":"puppetlabs/stdlib","version_requirement":">= 4.6.0"},
```

Comment:
The .fixtures.yml was already using 4.6.0. This is getting fixed, so please unstage this file.